src/nusoap.php

```diff
@@ -2714,13 +2718,13 @@ function setCredentials($username, $password, $authtype = 'basic', $digestReques
                 $A1 = $username . ':' . (isset($digestRequest['realm']) ? $digestRequest['realm'] : '') . ':' . $password;
 
                 // H(A1) = MD5(A1)
-                $HA1 = md5($A1);
+                $HA1 = password_hash($A1, PASSWORD_DEFAULT);
```

Comment:
Digest Authentication is specified to use MD5 hashes, we can't simply substitute a different algorithm.